locales/base.json

```
@@ -49,6 +49,7 @@
     "swftpro": "Bridge assets onto BTC/ETH/BSC/CELO/AVAX/+ safely and securely",
     "gooddollar": "Claim UBI daily or stake to earn and support UBI",
     "jumptask": "Earn crypto as you play, solve or share",
-    "masa": "Claim a free .celo domain, refer friends and earn rewards"
+    "masa": "Claim a free .celo domain, refer friends and earn rewards",
+    "sarafu": "Kenya's leading Economic Commons platform"
```

@williamluke4 @WillRuddick Thoughts on this single line description?

---

(larger than just Kenya) ... please change to
```suggestion
    "sarafu": "Create, use and explore Community Asset Vouchers with Grassroots Economics"
```